grpc/interceptor/logger.go

```diff
@@ -51,7 +49,12 @@ func Logger(l *logrus.Logger) grpc.UnaryServerInterceptor {
 	}
 }
 
-func addLoggerToContext(l *logrus.Logger, ctx context.Context) context.Context {
-	entry := l.WithField("request_id", uuid.NewV4().String())
+func addLoggerToContext(ctx context.Context, l *logrus.Logger) context.Context {
+	id, err := uuid.NewV4()
+	if err != nil {
+		logrus.WithError(err).Fatal("could not generate request id")
+	}
```

Comment:
This isn't ideal but I honestly have no idea why it would ever fail to generate 🤷‍♂️

Reply:
it relies on time, and the underlying time implementation can fail. i believe google's uuid has a wrapper w/ no error that will panic

Reply:
I figured that would be the error but the odds of that are so low that I'm cool with a crash. Somethings aren't recoverable.